app/views/appointments/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form_for(@appointment, url: appointments_path, data: { turbo: false }) do |f| %>
```

Comment:
Just curious: did you disable Turbo because you encountered issues submitting the form with it on or did you just copy from other forms in the project that don't use Turbo? (Totally fine, just wondering myself if Turbo is only causing issues with Devise or all forms)

Reply:
It wasn't working with Turbo on and I did notice you turned it off in the other file, so I sort of followed your lead. Not too sure what Turbo does though.